### PR TITLE
Don't Eat Trash

### PR DESCRIPTION
**Pull request by wylie**
adding some new transport SVGs :arrow_up_small: :arrow_forward: :arrow_down_small: :arrow_backward:

### CODE REVIEW

File: less/_typography.less [hunk 1]
```diff
@@ -87,6 +87,17 @@ body & {
 	color: @font-color;
 }
 
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
```
Comment (wylie) [inline less/_typography.less:95]: adding this in here because I can

**Comment by leachryanb** [comment]
Wow.  I mean, just, wow...

**Comment by wylie** [comment]
@leachryanb, I know, right?

File: less/_typography.less (hunk 1 above)
Comment (gjjones) [inline less/_typography.less:95]: That sounds like a good reason to me

**Pull request merged**